plugins/inputs/all/all.go

```diff
@@ -22,6 +22,7 @@ import (
 	_ "github.com/influxdata/telegraf/plugins/inputs/cgroup"
 	_ "github.com/influxdata/telegraf/plugins/inputs/chrony"
 	_ "github.com/influxdata/telegraf/plugins/inputs/cisco_telemetry_mdt"
+	_ "github.com/influxdata/telegraf/plugins/inputs/cisco_telemetry_mdt_netconf_dialin"
```

Comment:
Main thing I'm wondering is if this couldn't be added/integrated to the already existing plugin?

Reply:
It may end up being too dense of an implementation. As a first iteration, decoupling may be a good idea.

Reply:
Good, if you are done with your first iteration, you can merge them ;)

Reply:
Merge with master? :)

Reply:
No, merge with the existing plugin 😉

Reply:
The mechanism implemented for NETCONF is dial-in, making it more similar to the gNMI implementation. The original mdt plugin covers a dial-out mechanism for TCP and gRPC.

Would need to do an assessment on how much the implementations have in common.

Reply:
Oh okay, that can be kept separate indeed. Usually such kind of plugin is named with `_listener` at the end.